backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/functions/M_HU_Trace_Report.sql

```diff
@@ -18,14 +18,15 @@ AS
 $BODY$
 
 
-SELECT  t.LotNumber              AS LotNumber,
+SELECT t.LotNumber              AS LotNumber,
```

Comment:
wdyt about adding this DDL to de.metas.handlingunits as well?

Reply:
I moved this too but it's not visible here as "Outdated". I don't know why it is 🤔